requests-for-comments/2024-08-14-youtuber.md

```diff
@@ -0,0 +1,53 @@
+# Request for comments: Hire a YouTuber
+
+PostHog is built on content + word of mouth. YouTube has proven to be an effective channel for word of mouth grow, based on the significant number of people mention YouTube (or Theo) on signup, but we don’t have the expertise to do content right there. We currently differentiate by creating great content and YouTube is a massive opportunity to do that.
+
+YouTube rewards consistency and iteration. Our subscriber base and watch time continues to grow even though we don’t upload more videos. YouTube doesn’t work if you do it part time. You need to go all-in on video. 
+
+I think we have seen some successes, but have been too scattered to really capitalize on them:
+
+- Theo + YouTube mentions shows its promise
+- Although YouTube **tutorials** about PostHog don’t seem to work super well, topic-based (like blogs) YouTube videos do.
+- Although, anecdotally, my small number of YouTube videos get **more** mention than my blogs or newsletters from people, especially when they are learning PostHog. The complexity of PostHog lends itself to video. It is a great way to teach people how to use it.
+
+Beyond YouTube, what could a YouTuber do:
+
+- Create video social content + changelog like so many people have wanted to.
+- Experiment with short form content.
+- Collaborate with influencers (we have done this with newsletters and it works well). This would also help with my influencer plan.
+- Read out 20,000 names (just kidding… unless…).
```

Comment:
My only concern with this RFC is that we're not clear on the type of content we want this person to create, so Im concerned that they wont be set up for success. 
   
   I think to get them started, they can begin converting our newsletter issues to youtube videos. After a few months, they'll hopefully get a good feel for what kind of PostHog content works well and be able to drive it on their own